docs/privacy-policy.md

```diff
@@ -0,0 +1,172 @@
+information.
+
+When we believe it is necessary to prevent harm to you or someone else we may release 
+your non-personal information. We will only share your information in this way if we 
+have a good faith belief that it is reasonably necessary to protect the rights, 
+property or safety of you, our other users, Sempervirens, Numfocus or the public.
+
+How do we store and protect your information?
+-------------------------------------------------
+
+We are committed to never storing uniquely identifying personal information. 
+
+We will store your IP address, which is considered non-uniquely identifying personal 
+information. Non-uniquely identifying personal information will never be publicly 
+shared. Access to such data will be granted only to approved Sempervirens maintainers
+on an as-needed basis.
+
+We are committed to protecting your non-personal information once we have it. 
+We implement physical, business, and technical security measures as needed. 
+Despite our efforts, if we learn of a security breach, we'll notify you so that you 
+can take appropriate protective steps.
+
+How will we notify you?
+------------------------
+
+Since no uniquely identifying personal information is ever stored, no direct contact 
+with you will ever be initiated by us. Notifications will be issued via public 
+broadcasts such as via twitter, our website, or our mailing list.
+
+What else should you know?
+---------------------------
+
+We are a global organization and our computers are in several different places around 
+the world. We also use service providers whose computers may also be in various 
+countries. This means that your information might end  up on one of those computers 
+in another country, and that country may have a different level of data protection 
+regulation than yours. By giving us information, you consent to this kind of transfer 
+of your information. No matter what country your information is in, we comply with 
+applicable law and will also abide by the commitments we make in this privacy policy.
+
+If you are under 13, we do not want your information, and you must not 
+provide it to us. If you are a parent and believe that your child who is under 13 
+has provided us with personal or non-personal information, please contact us to have 
+the data removed. 
+
+What if we change this privacy policy or any of our privacy notices?
+----------------------------------------------------------------------
+
+We may need to change this policy and our notices. The updates will be posted online 
+via the broadcast notification channels listed above.
+
+Contact Us
+----------
+
+If you want to make a correction to your information, or you have any questions about 
+our privacy policies, please get in touch with:
+
+Numfocus
+....
+
+
+Document History
+------------------
+This document was forked from the Mozilla privacy policy: https://www.mozilla.org/en-US/privacy/
```

Comment:
According to the text at the bottom of that page, it is licensed CC BY-SA, meaning this document would need to be as well.